lib/task-data/tasks/bootstrap-ubuntu.js

```diff
@@ -7,12 +7,12 @@ module.exports = {
     injectableName: 'Task.Linux.Bootstrap.Ubuntu',
     implementsTask: 'Task.Base.Linux.Bootstrap',
     options: {
-        kernelFile: 'vmlinuz-3.16.0-25-generic',
-        initrdFile: 'initrd.img-3.16.0-25-generic',
+        kernelFile: 'vmlinuz-3.19.0-56-generic',
+        initrdFile: 'initrd.img-3.19.0-56-generic',
         kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
         initrdUri: '{{ api.server }}/common/{{ options.initrdFile }}',
-        basefs: 'common/base.trusty.3.16.0-25-generic.squashfs.img',
-        overlayfs: 'common/discovery.overlay.cpio.gz',
+        basefs: 'common/base.trusty.3.19.0-56-generic.squashfs.img',
+        overlayfs: 'common/discovery.3.19.0-56-generic.overlay.cpio.gz',
```

Comment:
How about to refactor the code using the task rendering? so it is more easy to change all micorkernel files to a new version by only change the options version:
```
options: {
        version: '3.19.0-56-generic',
        kernelFile: 'vmlinuz-{{options.version}}',
        initrdFile: 'initrd.img-{{options.version}}',
        kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
        initrdUri: '{{ api.server }}/common/{{ options.initrdFile }}',
        basefs: 'common/base.trusty.{{options.version}}.squashfs.img',
        overlayfs: 'common/discovery.{{options.version}}.overlay.cpio.gz',
        profile: 'linux.ipxe',
        comport: 'ttyS0'
    },
```